cookiecutter-django-app/{{cookiecutter.repo_name}}/Makefile

```
@@ -96,3 +99,7 @@ dummy_translations: ## generate dummy translation (.po) files
 build_dummy_translations: extract_translations dummy_translations compile_translations ## generate and compile dummy translation files
 
 validate_translations: build_dummy_translations detect_changed_source_translations ## validate translations
+
+install_transifex_client: ## Install the Transifex client
+	curl -o- https://raw.githubusercontent.com/transifex/cli/master/install.sh | bash
+	git checkout -- LICENSE README.md
```

What's the role of this line? It looks like it would undo local changes in those files, but I'm not sure why that would be needed.

---

We've moved from the old `python Transifex client` to new `Go Transifex client`. The new client installation also updates the `LICENSE` and `README.md` files but we don't need those changes to be pushed on the repo.

---

Oh, I see it now -- their script unpacks a tarball directly into the current directory, completely overwriting the LICENSE and README files. That's unfortunate.

My only concern here is that if someone has uncommitted changes to their README, they'll be silently dropped. I'm not sure if this is a real concern (is the client only installed from CI?) but I'd suggest adding this line above the client installation:

```
git diff -s --exit-code HEAD || { echo "Please commit changes first."; exit 1; }
```

It's probably also worth adding a minimal explanation of the `checkout` call, e.g.:
```suggestion
	git checkout -- LICENSE README.md # overwritten by Transifex installer
```

---

Transifex related changes will be done in the PR https://github.com/openedx/edx-cookiecutters/pull/196/files and the current PR will be rebased once that PR gets merged. 
The suggstions provided here have been included in the mentioned PR as well.